### DISCUSSION TITLE
mobile suggestions

### DISCUSSION THREAD

**Discussion by iveurne** [ppy/osu#30211] [Ideas]
lil dump
(im sorry if those have been suggested i just didnt see any like those while searching for similar things, i have very basic knowledge on how github discussion or github at all works)
- a send button for the typing section in chats because idk if im stupid but i think there isnt a way that you can send messages on the chat without a keyboard, usually you use the button in the bottom right to send messages but it does its other function that is make a new paragraph
- make the back button in the login & skin sections not close them and instead close the virtual keyboard first and then close the menu after a 2nd press, since in some ui scales the keyboard might hide elements like the skin you want to choose & the sign in button
- adjustable touch taiko drum controls (on width & height, take as an example taiko-san daijiro's drum size presets)
- adjustable mania touch controls with options to make it taller & less distracting by being able to adjust the opacity of the keys or even options for changing the style by making it separated by lines instead of translucent rounded squares
- adjustable buttons for ctb or just different styles of button layout presets
- the touch controls versatility suggestions could be added like how the skin editor currently works, but instead of skin elements its controls, or they could just be sliders in a new category, and maybe presets

**Comment by Tenexxt** [comment]
> a send button for the typing section in chats because idk if im stupid but i think there isnt a way that you can send messages on the chat without a keyboard, usually you use the button in the bottom right to send messages but it does its other function that is make a new paragraph

Yea it worked before, is a bug. Not sure when its going to get fixed since people are reporting this for a while now lol.

> adjustable touch controls

Yea its getting worked for on mania, not sure about taiko and ctb but i heard its also planned
